lib/v2/screens/receive_screens/receive_enter_data/receive_seeds_screen.dart

```diff
@@ -26,10 +28,18 @@ class ReceiveEnterDataScreen extends StatelessWidget {
               listenWhen: (_, current) => current.pageCommand != null,
               listener: (context, state) {
                 if (state.pageCommand is NavigateToReceiveDetails) {
-                  //Navigate to Receive Detail Page
+                  BlocProvider.of<ReceiveEnterDataBloc>(context)
+                      .add(const ClearPageState());
+                  NavigationService.of(context).navigateTo(
+                      Routes.receiveQR,
+                      ReceiveDetailArguments(
```

Comment:
create this ReceiveDetailArguments when the page command is created and pass it down with the page command. 
That way the view is cleaner.

Reply:
fix